bottlerocket/agents/src/tuf.rs

```diff
@@ -6,7 +7,29 @@ use url::Url;
 
 const ROOT_FILE_NAME: &str = "1.root.json";
 
-pub(crate) fn download_target(
+/// Parse both TUF urls for downloading OVAs
+pub fn tuf_repo_urls(
```

Comment:
Should this move to agent/utils?

Reply:
There are other TUF-related functions here in the `tuf` module that are used to interact with a TUF repository.

Reply:
I meant the entire module. I think we can keep it here for now, though.